content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -785,6 +785,65 @@ spec:
       storage: 10Gi
 ```
 
+## Volume Populators and Data Sources
+
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+
+{{< note >}}
+Volume populars are available as alpha since Kubernetes 1.18, are redesigned in Kubernetes 1.22. The `AnyVolumeDataSource` feature must be enabled. Refer to the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) documentation for more information.
```

Comment:
```suggestion
Volume populars are available as alpha since Kubernetes 1.18, and are redesigned in Kubernetes 1.22. The `AnyVolumeDataSource` feature must be enabled, to enable it please refer to the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) documentation for more information.
```

Reply:
BTW “populars” is a spelling error.